CHANGELOG.md

```diff
@@ -6,6 +6,22 @@
 
 - Changed subprocesses to execute without a shell.
 
+- Changed `-e` / `--exec` to consume all remaining arguments rather than a
+  single string. Each argument will be passed as a single argument to the
+  subprocess. It now must be the last argument in the command line. For example:
+
+  *Before*
+
+  ```console
+  parallel_test -e 'ruby -e "puts %[hello from process #{ENV[:TEST_ENV_NUMBER.to_s].inspect}]"' -n 5
```

Comment:
this seems more intuitive to me and it matches the --test-options behavior

Reply:
can we do shellsplit instead ?

Reply:
Works for me. This is done in the latest revision. Let me know if you'd like me to squash commits.